### ISSUE TITLE
incompatible with the columns-plugin using shortcode commands

### ISSUE THREAD

**Issue by GertVanKruiningen** [picandocodigo/List-Category-Posts#196]
I am using two plugins with my GeneratePress theme.

Your plugin **list-category-post** works fine, and is listing all posts of a certain category very nicely on a page.

I am now also using the **columns plugin** to create columns using shortcode commands. When editing and viewing a single post using the **columns** plugin, that works fine, its creating columns for me when I view the single post on its own.

BUT when I look at the page that has the posts listed using list-category-posts, the the posts that are using the columns formatted using the column plugin with the shortcode commands are empty. Its almost like the list-category-post plugin doesn't recognize the columns. If I remove the column shortcode commands I see the text or photos again.

Is there a way I can debug this, or has anyone seen this before and has a solution for this please.

Just looking through known issues or features, I noticed #47 "allow shortcodes to work in excerpts" and I am wondering if that is the reason the columns pluging doesn't function.
By the way I have used other plugins to do the columns formating using shortcodes, and all of them fail, so yes it could be this #47 issue.

Looking forward to some feedback

Regards
Gert van Kruiningen
New Zealand

**Comment by GertVanKruiningen** [comment]
I also just noticed, and this could be the same issue again as what I described above: the plugin "Simple YouTube Embed" doesn't work. This plugin allows you to enter a YouTube URL in the text of a post or a page and it creates a YouTube window.
The same as above here.  Looking at the post on its own, it works fine. But if the post is listed on a page with the list-category-post plugin, its not there, you just see the YouTube URL.
And the plugin works fine on a page. So all the same symptoms as the ones I described above.

It would be so good if this issue could be resolved. 
I would help with it if I had the knowledge to do it, but I don't, so hopefully someone else has...

Regards
Gert
